CHANGES.rst

```diff
@@ -47,6 +47,7 @@ Bugfixes
 - Fix paper reviewers getting notifications even if their type of reviewing
   has been disabled (:issue:`3852`)
 - Correctly handle merging users in the paper reviewing module (:issue:`3895`)
+- Show correct number of registrations (:issue:`#3935`)
```

Comment:
without the `#`

Reply:
"....in management area" ?

Reply:
It is not only management area. There is one place in the display area where we show the number of registrations.